.github/ISSUE_TEMPLATE.md

```diff
@@ -43,6 +46,8 @@ Please post as much of your database schema as necessary. If you are using `infe
 - [ ] I have already looked over the [issue tracker](https://github.com/diesel-rs/diesel/issues) for similar issues.
 - [ ] This issue can be reproduced on Rust's stable channel. (Your issue will be
   closed if this is not the case)
+- [ ] This issue is not a feature request, or I was asked to open it by a member
+  of the Diesel team.
```

Comment:
Maybe add something like "(please ask for features on discourse.diesel.rs instead)" here, too for people who skipped directly to the checkboxes